mlir/test/Target/LLVMIR/Import/tail-kind.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+; CHECK: llvm.func @tailkind()
+declare void @tailkind()
+
+; CHECK-LABEL: @call_tailkind
+define void @call_tailkind() {
+  ; CHECK: llvm.call musttail @tailkind()
+  musttail call void @tailkind()
+
```

Comment:
```suggestion
```
ultra nit: I would drop these newlines before the return here and in the tests below.